docs/BENCHMARKS.md

```diff
@@ -5,12 +5,12 @@
 1. Benchmarks are run by JUnit as a separate source set just like main, test and integration-test. The benchmarks can be 
 found [here](/src/benchmark/java/com/commercetools/sync/benchmark).
 
-2. Every benchmark writes it's results as JSON to 
-[benchmarks.json](https://commercetools.github.io/commercetools-sync-java/benchmarks/benchmarks.json) which is saved in
-the gh-pages branch of the repo.
+2. Every time a commit is made in any branch, the benchmarks are run and the run shows whether the benchmarks exceed the
+pre-defined threshold, so that we know if the corresponding commit influences the performance.
 
-3. Every time a release is made or the build has a git tag, the benchmarks are run and the results are added to 
-[benchmarks.json](https://commercetools.github.io/commercetools-sync-java/benchmarks/benchmarks.json).
+3. Benchmark results are written as JSON to 
+[benchmarks.json](https://commercetools.github.io/commercetools-sync-java/benchmarks/benchmarks.json) only when a 
+release is made or the build has a git tag. The JSON file is saved in the gh-pages branch of the repo.
```

Comment:
What's the difference between release is made and the build has a git tag? How do you make a release?

You can also link to the branch `gh-pages`.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/868/commits/74c54b584bcef747a68db33bd78dd41ac975b5f0